CHANGES.rst

```diff
@@ -1,3 +1,25 @@
+pywb 2.3.x changelist
+~~~~~~~~~~~~~~~~~~~~~
+
+* Wombat Improvements and spin-of:
```

Comment:
I think `spin-off` does not correctly convey why we pulled wombat out of pywb where as `modularization` would.

```suggestion
* Wombat Improvements and modularization:
```